src/app.js

```diff
@@ -102,25 +103,65 @@ go.app = function() {
                             section_name : choice.label,
                             section_id : choice.value,
                             opts_data : opts.data,
-                            location_name : opts.full_name
+                            location_name : opts.full_name,
+                            location_id : opts.full_geoid
                         }
                     };
                 }
             });
         });
 
+        function sub_section(data, section_id) {
+            return sub_section[section_id](data);
+        }
+
+        sub_section.elections = function(data) {
+            //I would like to loop through the party distribution to return the top 3 parties
+            return data.provincial_2014.name + ":\n" + "Registered voters = " + data.provincial_2014.registered_voters.values.this + "\n" + data.provincial_2014.average_turnout.values.this + "% cast their vote";
```

Comment:
I think we can make these functions more readable by using the `join` trick to lay the lines out nicely, e.g.
```javascript
return [
    data.provincial_2014.name + ":",
    "Registered voters = " + data.provincial_2014.registered_voters.values.this,
    data.provincial_2014.average_turnout.values.this + "% cast their vote",
].join("\n");
```